spec/controllers/url_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe UrlsController do
+  describe 'POST create' do
+    let(:valid_params) { {'url' => {'url_orginal' => 'http://wkostanski.pl'}} }
+    let(:invalid_params) { {'url' => {'url_orginal' => 'wkostanski.pl'}} }
+    it 'with valid params' do
+      expect do
+        post :create, params: valid_params
```

Comment:
This line is duplicated across 3 tests therefore you can reduce the duplication by moving it to a subject, e.g 
```ruby
subject { post :create, params: valid_params }

it 'something' do
  expect(subject).to change(....)
end
```